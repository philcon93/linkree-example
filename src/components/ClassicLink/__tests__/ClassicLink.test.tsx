import { render } from '@testing-library/react';
import { ClassicLink } from '../';

describe('ClassicLink', () => {
    test('Displays correctly', () => {
        const { container, getByText } = render(
            <ClassicLink title={'title text'} />
        );

        expect(container).toMatchSnapshot();
        expect(getByText('title text')).toBeInTheDocument();
    });
});
