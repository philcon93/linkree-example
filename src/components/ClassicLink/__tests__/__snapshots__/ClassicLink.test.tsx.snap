// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClassicLink Displays correctly 1`] = `
<div>
  <a
    class="chakra-link css-1huwrpe"
    rel="noopener noreferrer"
    target="_blank"
  >
    <span>
      title text
    </span>
  </a>
</div>
`;
